optuna/visualization/matplotlib/_utils.py

```diff
@@ -44,3 +44,12 @@ def _is_categorical(trials: List[FrozenTrial], param: str) -> bool:
         for t in trials
         if param in t.params
     )
+
+
+def _is_numerical(trials: List[FrozenTrial], param: str) -> bool:
```

Comment:
Why don't you use `optuna.visualization._utils._is_numerical`?

Reply:
I just followed the definition of `_is_categorical` in current code base: same `_is_categorical` is deifned in both `optuna.visualization._utils.py` and `optuna/visualization/matplotlib/_utils.py`.

Reply:
Oh, I see. In that case, LGTM! Although I do not know the context, we may remove them in another PR after some discussion.